src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -65,7 +73,14 @@ export default class GroupComparisonPage extends React.Component<
     @observable.ref private store: GroupComparisonStore;
     private queryReaction: IReactionDisposer;
     private urlWrapper: GroupComparisonURLWrapper;
-
+    clearMessage: (() => void) | undefined;
```

Comment:
these new properties are not tagged with @observable decorator.  It would thus surprise me if updating them caused the interface to re-render.